### ISSUE TITLE
Bug in "lsm6dsm_fifo_write_trigger_get".

### ISSUE THREAD

**Issue by qutefox** [STMicroelectronics/lsm6dsm#1]
The method "lsm6dsm_fifo_write_trigger_get" (in lsm6dsm_reg.c) reads both LSM6DSM_FIFO_CTRL2 and LSM6DSM_MASTER_CONFIG registers.
But it only uses the LSM6DSM_FIFO_CTRL2 , twice.

I beleive the code:
```
switch ((fifo_ctrl2.timer_pedo_fifo_drdy << 1) +
            fifo_ctrl2. timer_pedo_fifo_drdy)
```
should be:
```
switch ((fifo_ctrl2.timer_pedo_fifo_drdy << 1) +
            master_config. data_valid_sel_fifo )
```

**Comment by qutefox** [comment]
Actually looking at this enum
```
typedef enum
{
  LSM6DSM_TRG_XL_GY_DRDY     = 0,
  LSM6DSM_TRG_STEP_DETECT    = 1,
  LSM6DSM_TRG_SH_DRDY        = 2,
} lsm6dsm_trigger_fifo_t;
```
and "lsm6dsm_fifo_write_trigger_set"
it should be the other way around:
```
switch ((master_config. data_valid_sel_fifo << 1) +
            fifo_ctrl2.timer_pedo_fifo_drdy)
```

**Comment by avisconti** [comment]
Hello @qutefox, thanks very much! Can you provide a PR with two separate commits addressing this issue? It would be quite helpful for me and would speed up the fixing.

**Comment by qutefox** [comment]
PR as requested: https://github.com/STMicroelectronics/lsm6dsm-pid/pull/3

**Comment by avisconti** [comment]
Fixed by #3
